src/features/colinks/Mutes.tsx

```diff
@@ -20,12 +20,13 @@ export const Mutes = ({
   targetProfileId: number;
   targetProfileAddress: string;
 }) => {
-  const profileId = useAuthStore(state => state.profileId);
+  const profileId = useProfileId(false);
```

Comment:
we just shouldn't show this whole component if not logged in

Reply:
hmm, lets discuss.